src/styles/index.scss

```diff
@@ -1,3 +1,92 @@
+$square-background: #eee;
+$square-size: 100px;
+$square-border: 1px solid black;
+$square-text-size: 30px;
+$days-in-month: 31;
+$square-margin: 1px;
+$square-text-family: Arial, Helvetica, sans-serif;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+ul,
+li {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.calendar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar__days {
+  display: flex;
+  flex-wrap: wrap;
+  width: 707px;
+  list-style-type: decimal;
+  transition-duration:
+    linear 0,
+    5s;
+
+  .calendar__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    background: $square-background;
+    width: $square-size;
+    height: $square-size;
+    border: $square-border;
+    margin-bottom: $square-margin;
+    margin-right: $square-margin;
+    gap: $square-margin;
+    text-align: center;
+    font-size: $square-text-size;
+    font-family: $square-text-family;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
Lucas, you have removed the hovering styles that you previously had and that were asked by the task's checklist.
Please fix it to finish this task. ONLY change this, the rest is correct